src/main/java/com/imaee/propinq/properties/services/implementations/PropertyTypeService.java

```diff
@@ -0,0 +1,100 @@
+package com.imaee.propinq.properties.services.implementations;
+
+import com.imaee.propinq.properties.controllers.requests.PropertyTypeRequest;
+import com.imaee.propinq.properties.controllers.responses.PropertyTypeResponse;
+import com.imaee.propinq.properties.data.models.PropertyType;
+import com.imaee.propinq.properties.data.repositories.IPropertyTypeRepository;
+import com.imaee.propinq.properties.mappers.PropertyTypeMapper;
+import com.imaee.propinq.properties.services.interfaces.IPropertyTypeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class PropertyTypeService implements IPropertyTypeService {
+
+    @Autowired
+    private IPropertyTypeRepository propertyTypeRepository;
+
+    private final String MSG_ALREADY_EXISTS = "PROPERTYTYPE WITH THIS NAME ALREADY EXISTS";
+    private final String MSG_NOT_EXISTS = "PROPERTYTYPE WITH THIS ID NOT EXISTS";
+
+    @Override
+    public List<PropertyTypeResponse> getPropertyType() {
+        List<PropertyType> propertyTypes = propertyTypeRepository.findByState(PropertyType.ACTIVE);
+        return propertyTypes.stream().map(PropertyTypeMapper::toPropertyTypeResponse).toList();
+    }
```

Comment:
Muy bien teniendo en cuenta el estado!

Quizás es medio innecesario reservar el espacio de memoria de la variable ya que la devolvés inmediatamente después

```suggestion
    @Override
    public List<PropertyTypeResponse> getPropertyType() {
        return propertyTypeRepository.findByState(PropertyType.ACTIVE).stream()
            .map(PropertyTypeMapper::toPropertyTypeResponse)
            .toList();
    }
```